skyreader/result.py

```diff
@@ -1132,7 +1094,7 @@ def area(vs):
                 c='r', linestyle='dashed', label=contour_label)
 
         # Output contours in RA, dec instead of theta, phi
-        saving_contours = []
+        saving_contours: list = []
```

Comment:
This isn't needed unless you subscript a sub-type

Reply:
I think `mypy` was complaining here.

Reply:
```
skyreader/result.py:1097: error: Need type annotation for "saving_contours" (hint: "saving_contours: List[<type>] = ...")  [var-annotated]
```

Reply:
odd... but okay if mypy says so!